### PR TITLE
add cloning instructions to readme

### PR DESCRIPTION
**Pull request by amsnyder**
Adding some basic instructions to clone the repo and get started. I kept it pretty basic, so feel free to add more flare if you'd like. I mostly just wanted to get something up to hand over to David.

Once we merge these instructions, I think we need to:
- copy over the readme and issue_drafts to snakemake-introduction (the real starting point, this is just the answer key - in fact, maybe we should delete them from here so we only have one copy to maintain)
- ask David to test out the lesson, given the snakemake-introduction repository
- figure out if we want to convert the md files into issues, and update the instructions if we do

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# snakemake-tutorial-scratch
\ No newline at end of file
+# snakemake-tutorial-scratch
+This repository was built to be used as an introductory tutorial for using Snakemake as a pipelining tool for Python code. It introduces some general Snakemake concepts and gives an opportunity to practice implementing them on a small example problem.
+
+The tutorial is broken up into an ordered set of lessons, each of which introduces a new Snakemake concept (or sometimes two). Each lesson is contained in a markdown file in the `issue_drafts` folder of this repository. The rest of the code contained in the repository is the codebase you will start from and modify as you go through each lesson.
+
+In order to walk through this tutorial, you should clone this repository onto your local computer, and open up the markdown file `issue_drafts/issue_0.md`, which takes you through some basic computer setup. You can then proceed through the following issue drafts to continue through the tutorial.
```
Comment (amsnyder) [inline README.md:6]:
```suggestion
In order to walk through this tutorial, you should make a fork of this repository onto your own Github. Then you can clone your fork of this repository onto your local computer to work on.

To begin the tutorial, open up the markdown file `issue_drafts/issue_0.md`, which takes you through some basic computer setup. You can then proceed through the following issue drafts to continue through the tutorial.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
-# snakemake-tutorial-scratch
\ No newline at end of file
+# snakemake-tutorial-scratch
+This repository was built to be used as an introductory tutorial for using Snakemake as a pipelining tool for Python code. It introduces some general Snakemake concepts and gives an opportunity to practice implementing them on a small example problem.
+
+The tutorial is broken up into an ordered set of lessons, each of which introduces a new Snakemake concept (or sometimes two). Each lesson is contained in a markdown file in the `issue_drafts` folder of this repository. The rest of the code contained in the repository is the codebase you will start from and modify as you go through each lesson.
+
+In order to walk through this tutorial, you should make a fork of this repository onto your own Github. Then you can clone your fork of this repository onto your local computer to work on.
+
+To begin the tutorial, open up the markdown file `issue_drafts/issue_0.md`, which takes you through some basic computer setup. You can then proceed through the following issue drafts to continue through the tutorial.
```
Comment (amsnyder) [inline README.md:8]:
```suggestion
To begin the tutorial, open up the markdown file `issue_drafts/issue_0.md`, which takes you through some basic computer setup. You can then proceed through the following issue drafts to continue through the tutorial.

For every issue (except issue 0), you will be making changes to the code in the repository. If you are working with an instructor for this tutorial, you should push your changes from you local repository to your remote repository at the after finishing each issue. You can then open a pull request on your fork of the repository with your code changes, and request review by your instructor. After they approve the pull request, make sure to pull down and changes from Github to your local copy of the repository, and continue on with the next issue.
```

**Review by AndyMcAliley — APPROVED** [review]
Sounds good! At some point, we can rename the folder "issue_drafts" to "issues".

**Comment by AndyMcAliley** [comment]
Or we could call them "lessons" instead of "issues", if we're not going to convert them into issues anyway.

**Comment by amsnyder** [comment]
> Or we could call them "lessons" instead of "issues", if we're not going to convert them into issues anyway.

I like lessons! I'll update the folder name

**Comment by amsnyder** [comment]
> > Or we could call them "lessons" instead of "issues", if we're not going to convert them into issues anyway.
> 
> I like lessons! I'll update the folder name

Actually - let's wait until we have David review everything and decide what format they will be (will they actually get transformed into issues?). I'll make the folder name just "issues" for now

**Pull request merged**